rel/config.exs

```diff
@@ -0,0 +1,53 @@
+# Import all plugins from `rel/plugins`
+# They can then be used by adding `plugin MyPlugin` to
+# either an environment, or release definition, where
+# `MyPlugin` is the name of the plugin module.
+Path.join(["rel", "plugins", "*.exs"])
+|> Path.wildcard()
+|> Enum.map(&Code.eval_file(&1))
+
+use Mix.Releases.Config,
+    # This sets the default release built by `mix release`
+    default_release: :default,
+    # This sets the default environment used by `mix release`
+    default_environment: Mix.env()
+
+# For a full list of config options for both releases
+# and environments, visit https://hexdocs.pm/distillery/configuration.html
+
+
+# You may define one or more environments in this file,
+# an environment's settings will override those of a release
+# when building in that environment, this combination of release
+# and environment configuration is called a profile
+
+environment :dev do
+  # If you are running Phoenix, you should make sure that
+  # server: true is set and the code reloader is disabled,
+  # even in dev mode.
+  # It is recommended that you build with MIX_ENV=prod and pass
+  # the --env flag to Distillery explicitly if you want to use
+  # dev mode.
+  set dev_mode: true
+  set include_erts: false
+  set cookie: :"pDlr~SGoSLjoW12Z}hefp7vH!nN[0?Pj=,W6:>fkB/Hvjd=6)uE<*JiZ~E5(DpK7"
+end
+
+environment :prod do
+  set include_erts: true
+  set include_src: false
+  set cookie: :"l^m/AL2P4|/~aF)&j|j8<PT1bRuAA*kYEA`8L0XQuh=5NVkuu%.0^PVPXBVgtKFL"
```

Comment:
Security concern: Erlang magic cookie shouldn't be hardcoded. Probably take from `dotenv` instead

This is probably a good reference: https://hackernoon.com/mastering-elixir-releases-with-distillery-a-pretty-complete-guide-497546f298bc